src/components-examples/material/stepper/stepper-harness/stepper-harness-example.spec.ts

```diff
@@ -0,0 +1,89 @@
+import {TestBed, ComponentFixture, waitForAsync} from '@angular/core/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import {MatStepperHarness, MatStepperNextHarness} from '@angular/material/stepper/testing';
+import {HarnessLoader, parallel} from '@angular/cdk/testing';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting}
+  from '@angular/platform-browser-dynamic/testing';
+import {MatStepperModule} from '@angular/material/stepper';
+import {StepperHarnessExample} from './stepper-harness-example';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {ReactiveFormsModule} from '@angular/forms';
+import {STEPPER_GLOBAL_OPTIONS} from '@angular/cdk/stepper';
+
+describe('ToolbarHarnessExample', () => {
+  let fixture: ComponentFixture<StepperHarnessExample>;
+  let loader: HarnessLoader;
+
+  beforeAll(() => {
+    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+  });
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [MatStepperModule, NoopAnimationsModule, ReactiveFormsModule],
+        declarations: [StepperHarnessExample],
+        providers: [{
+          provide: STEPPER_GLOBAL_OPTIONS,
+          useValue: {showError: true} // Required so the error state shows up in tests.
```

Comment:
I think this part is mostly relevant for our own tests and can be removed from here together with the error state test below.